config.json.example

```diff
@@ -50,6 +50,7 @@
             "consumerSecret": "change this"
         },
         "github": {
+            "enterpriseURL": "if you use github.com, not need to specify",
```

Comment:
I think this line should be removed because we expect user copy this file as `config.js`. User will have to remove this line in config manually.

The comment part could be added in `lib/config/default.js` instead. 

@jackycute Just a reminder, we’ll also have to update our online documentation accordingly.

Reply:
thanks! it's fixed.